Runtime/WebGLRedisMessagingClient.cs

```diff
@@ -0,0 +1,430 @@
+
+        private static Subject<(WebGLJoinResponse joinResponse, string instanceId)> onJoinMessageReceived;
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void ReceiveJoinResponse(string joinResponse, string instanceId)
+            => onJoinMessageReceived.OnNext((JsonSerializer.Deserialize<WebGLJoinResponse>(joinResponse), instanceId));
+
+        private static Subject<(bool isConnected, string instanceId)> onJoiningGroupStatusReceived;
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void HandleJoiningGroupStatus(string isConnected, string instanceId)
+            => onJoiningGroupStatusReceived.OnNext((bool.Parse(isConnected), instanceId));
+
+        private static Subject<(string reason, string instanceId)> onLeavingEventReceived;
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void HandleOnLeaving(string reason, string instanceId)
+            => onLeavingEventReceived.OnNext((reason, instanceId));
+
+        private static Subject<(string reason, string instanceId)> onUnexpectedLeftEventReceived;
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void HandleOnUnexpectedLeft(string reason, string instanceId)
+            => onUnexpectedLeftEventReceived.OnNext((reason, instanceId));
+
+        private static Subject<(string userId, string instanceId)> onUserJoinedEventReceived;
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void HandleOnUserJoined(string userId, string instanceId)
+            => onUserJoinedEventReceived.OnNext((userId, instanceId));
+
+        private static Subject<(string userId, string instanceId)> onUserLeavingEventReceived;
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void HandleOnUserLeaving(string userId, string instanceId)
+            => onUserLeavingEventReceived.OnNext((userId, instanceId));
+
+        private static Subject<(Message message, string instanceId)> onMessageReceivedEventReceived;
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void HandleOnMessageReceived(string message, string instanceId)
+            => onMessageReceivedEventReceived.OnNext((JsonSerializer.Deserialize<Message>(message), instanceId));
+
+        private static Subject<string> onStopSocketCalled;
+        [MonoPInvokeCallback(typeof(Action<string, string>))]
+        private static void StopSocket(string instanceId, string unused)
+            => onStopSocketCalled.OnNext(instanceId);
+
+        protected override void DoReleaseManagedResources()
+        {
+            cancellation.Dispose();
+            WebGLHelper.CallAction(WithPrefix(nameof(DoReleaseManagedResources)), instanceId);
+
+            instanceNum--;
+            if (instanceNum == 0)
+            {
+                disposables.Dispose();
+            }
```

Comment:
disposablesはこの外でDisposeし、この中ではStaticDisposablesがDisposeされるべきです。
```suggestion
            disposables.Dispose();

            instanceNum--;
            if (instanceNum == 0)
            {
                StaticDisposables.Dispose()
            }
```

Reply:
指摘通りに対応しました。
[e0ce708](https://github.com/extreal-dev/Extreal.Integration.Messaging.Redis/pull/1/commits/e0ce70874c749e60190ccec1ee6ef48e63fa4fa1)

Reply:
確認しました。